### DISCUSSION TITLE
TeleHit [MX]

### DISCUSSION THREAD

**Discussion by Carlinhos027** [iptv-org/community#386] [Show and tell]
### Channel ID

TeleHit.mx

### Stream URL

https://linear-417.frequency.stream/417/hls/master/playlist.m3u8

### Notes

Telehit Musica: https://linear-418.frequency.stream/418/hls/master/playlist.m3u8
Plus: https://linear-419.frequency.stream/419/hls/master/playlist.m3u8

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

**Comment by freearhey** [comment]
https://linear-417.frequency.stream/417/hls/master/playlist.m3u8 (offline)
<img width="455" alt="image" src="https://user-images.githubusercontent.com/7253922/232256649-de883e54-97cb-434f-8cf9-2a0c1b47c587.png">
